qiita_db/handlers/artifact.py

```diff
@@ -69,6 +69,46 @@ def get(self, artifact_id):
 
         self.write(response)
 
+    @authenticate_oauth
+    def patch(self, artifact_id):
+        """Patches the filepaths of the artifact
+
+        Parameter
+        ---------
+        artifact_id : str
+            The id of the artifact whole filepaths information is being updated
+
+        Returns
+        -------
+        dict
+            Format:
+            {'success': bool,
```

Comment:
Good point. The difference between here and the other dictionaries is that this is part of the REST api that we put in place for the plugins, while the other ones are used in the GUI. It turns out that we did not catch those differences when we reviewed the GUI code adding this structure.

If you feel strong about it I can change but I will need to change all the qiita_db handlers plus the plugins consuming that API. I would prefer to not change it right now, given that we should end up changing this entire API to not return the success keyword but rather raising an HTTP error in case the call is not successful. Does it make sense?